lib/request_handler/base.rb

```diff
@@ -96,11 +96,14 @@ def parse_options(type:, parser:)
 
     def parse_body_params
       defaults = fetch_defaults('body.defaults', {})
-      defaults.merge(BodyParser.new(
-        request:        request,
-        schema:         lookup!('body.schema'),
-        schema_options: execute_options(lookup('body.options'))
-      ).run)
+      parsed_body = BodyParser.new(
+        request:          request,
+        schema:           lookup!('body.schema'),
+        schema_options:   execute_options(lookup('body.options')),
+        included_schemas: lookup('body.included')
+      ).run
+      return defaults.merge(parsed_body) unless parsed_body.is_a? Array
```

Comment:
hmm I think I'd rather drop the support for default body then this - Not sure why we added this in the first place and if it even makes sense to have a default for that.